src/v3/test/integration/flow-verify-with-piv-as-authenticator.test.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import selectAuthenticatorListWithPIV from '../../src/mocks/response/idp/idx/authenticator-select-piv.json';
+import pivVerifyResponse from '../../src/mocks/response/idp/idx/authenticator-piv-verify.json';
+import { createAuthJsPayloadArgs, setup } from './util';
+
+describe('flow-verify-with-piv-as-authenticator', () => {
+  it('should clear global errors when transitioning between pages', async () => {
```

Comment:
I think this test title needs to be changed

Reply:
Great catch, I was doomed by the copy and past monster 😅